src/locales/uk/parts/admin.ts

```diff
@@ -46,8 +46,7 @@ export const admin = {
   changeDateSuccess: 'Дату публікації змінено',
   scheduleDateSuccess: 'Дату публікації заплановано',
   schedulePublishTitle: 'Запланувати публікацію?',
-  from: 'з',
```

Comment:
it's not used anywhere else, is it?

Reply:
No, this code isn't used anywhere. I searched in the project before I removed this line.